UnityProject/Assets/IngameDebugConsole/Scripts/DebugLogUnitystationCommands.cs

```diff
@@ -29,10 +30,36 @@ public class DebugLogUnitystationCommands : MonoBehaviour
 	{
 		private static bool IsAdmin()
 		{
-			if (CustomNetworkManager.IsHeadless) return false;
-			if (AdminCommandsManager.IsAdmin(PlayerManager.LocalPlayerScript.PlayerInfo.Connection, out _) != false) return true;
-			Logger.Log("This function can only be executed by admins.", Category.DebugConsole);
-			return false;
+			try
+			{
+				if (CustomNetworkManager.IsHeadless)
+				{
+					Logger.Log("Will not execute this function on a headless server.");
+					return false;
+				}
+				if (AdminCommandsManager.IsAdmin(PlayerManager.LocalPlayerScript.PlayerInfo.Connection, out _) != false) return true;
+				Logger.Log("This function can only be executed by admins.", Category.DebugConsole);
+				return false;
+			}
+			catch (Exception e)
+			{
+				var report = new StringBuilder();
+				report.AppendLine("Attempted to check for local player if they're an admin but something went wrong..");
+				report.AppendLine("-----------");
+				report.AppendLine($"AdminCommandsManager not null: {AdminCommandsManager.Instance != null}");
+				report.AppendLine($"Player Manager not null: {PlayerManager.Instance != null}");
+				if (PlayerManager.Instance != null)
+				{
+					report.AppendLine($"Player Script not null: {PlayerManager.LocalPlayerScript != null}");
+					report.AppendLine($"Player Info not null: {PlayerManager.LocalPlayerScript.PlayerInfo != null}");
+					report.AppendLine($"Player connection not null: {PlayerManager.LocalPlayerScript.PlayerInfo?.Connection != null}");
+				}
+				report.AppendLine("-------");
+				report.AppendLine(e.ToString());
+				Logger.LogError(report.ToString());
+				// To avoid disruption, allow IsAdmin() to continue normally; the server will handle anti-cheat in it's own functions.
+				return true;
```

Comment:
shouldn't this return false?

Reply:
Read the comment

Reply:
`allow IsAdmin() to continue normally; the server will handle anti-cheat in it's own functions`

Reply:
Did you trigger the debug command in the lobby? That would mean LocalPlayerScript would be null, you'd want LocalViewerScript (think thats what it is called for player viewer script) when in lobby.

Reply:
No, while in a round and in a character